src/features/user/Account/CancelModal.tsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
-import Modal from '@material-ui/core/Modal';
-import Backdrop from '@material-ui/core/Backdrop';
-import Fade from '@material-ui/core/Fade';
-import FormControl from '@material-ui/core/FormControl';
-import RadioGroup from '@material-ui/core/RadioGroup';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
 import { ThemeContext } from '../../../theme/themeContext';
 import styles from './AccountHistory.module.scss';
 import { useTranslation } from 'react-i18next';
 import { putAuthenticatedRequest } from '../../../utils/apiRequests/api';
 import { UserPropsContext } from '../../common/Layout/UserPropsContext';
 import GreenRadio from '../../common/InputTypes/GreenRadio';
-import { ThemeProvider } from '@material-ui/styles';
-import { Calendar, MuiPickersUtilsProvider } from '@material-ui/pickers';
-import DateFnsUtils from '@date-io/date-fns';
 import materialTheme from '../../../theme/themeStyles';
 import Close from '../../../../public/assets/images/icons/headerIcons/close';
 import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
-import { CircularProgress } from '@material-ui/core';
+import {
+  CircularProgress,
+  Modal,
```

Comment:
There are minor changes seen in spacing/font styles within the CancelModal.

Reply:
![Screenshot 2022-05-05 at 5 22 03 PM](https://user-images.githubusercontent.com/44917347/166917625-fbaf3156-9f8d-47c7-a2d0-427cb347c436.png)

Reply:
@mohitb35 Please verify the zoom level
The left window seems to e more zoomed-in than right

Reply:
Fixed in #1467